pkg/transformers/greenmask/greenmask_unix_timestamp_transformer.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package greenmask
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/xataio/pgstream/pkg/transformers"
+)
+
+var errMinMaxValueNotSpecified = errors.New("greenmask_unix_timestamp: min_value and max_value must be specified")
+
+func NewUnixTimestampTransformer(generatorType transformers.GeneratorType, params transformers.Parameters) (*IntegerTransformer, error) {
+	minValue, foundMin, err := transformers.FindParameter[int64](params, "min_value")
+	if err != nil {
+		return nil, fmt.Errorf("greenmask_unix_timestamp: min_value must be an int64: %w", err)
+	}
+
+	maxValue, foundMax, err := transformers.FindParameter[int64](params, "max_value")
+	if err != nil {
+		return nil, fmt.Errorf("greenmask_unix_timestamp: max_value must be an int64: %w", err)
+	}
+
+	if !foundMin || !foundMax {
+		return nil, errMinMaxValueNotSpecified
+	}
+
+	transformer, err := NewIntegerTransformer(generatorType, transformers.Parameters{
```

Comment:
Could we not use the params on input, add the size and leave the validation to the integer transformer? I think right now the min/max values are validated twice.